parinfer-rust-mode.el

```diff
@@ -551,7 +551,7 @@ not available."
 ;;;###autoload
 (define-minor-mode parinfer-rust-mode
   "A simpler way to write lisps."
-  :lighter " parinfer"
+  :lighter (:eval (concat " parinfer:" parinfer-rust--mode))
```

Comment:
Nice I learned about eval today :)

This is such a simple fix that I never thought about because I never use "lighters". Thanks!